test-network-function/common/env.go

```diff
@@ -44,7 +44,7 @@ var DefaultTimeout = time.Duration(defaultTimeoutSeconds) * time.Second
 
 // GetContext spawns a new shell session and returns its context
 func GetContext() *interactive.Context {
-	context, err := interactive.SpawnShell(interactive.CreateGoExpectSpawner(), DefaultTimeout, interactive.Verbose(true))
+	context, err := interactive.SpawnShell(interactive.CreateGoExpectSpawner(), DefaultTimeout, interactive.Verbose(true), interactive.SendTimeout(DefaultTimeout*time.Second))
```

Comment:
What about the other two contexts (SpawnOc/SpawnSSH)?

Reply:
I think the "*time.Second" is not needed here, since it was already done to the DefaultTimeout var.

Reply:
right. I'll fix it now